AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/editor_component/editor_script_canvas.py

```diff
@@ -0,0 +1,119 @@
+
+We are deprecating this file and moving the constants to area specific files. See the EditorPythonTestTools > consts
+folder for more info
+"""
+from editor_python_test_tools.editor_entity_utils import EditorEntity, EditorComponent
+import azlmbr.scriptcanvas as scriptcanvas
+from enum import Enum
+import azlmbr.math as math
+from consts.scripting import (SCRIPT_CANVAS_UI)
+import azlmbr.bus as bus
+import azlmbr.editor as editor
+import azlmbr.entity as entity
+import azlmbr.legacy.general as general
+import azlmbr.object
+
+
+class VariableState(Enum):
+    UNUSEDVARIABLE = 0
+    VARIABLE = 1
+
+
+class Path:
+
+    # component property path
+    SCRIPT_CANVAS_ASSET_PROPERTY = "Configuration|Source"
+
+    # path parts
+    PROPERTIES = "Configuration|Properties|"
+    UNUSED_VARIABLES = "Unused Variables|"
+    VARIABLES = "Variables|"
+    DATUM_PATH = "Datum|Datum|value|"
+
+    def construct_variable_component_property_path(self, variable_name: str, variable_state: VariableState) -> str:
+        """
+        Function for constructing a component property path for the value setting function
+
+        param variable_name: the name of the variable being assigned
+        param variable_state: used/unused variable
+
+        returns a string for the component property path
+        """
+
+        component_property_path = self.PROPERTIES
+
+        match variable_state:
+            case VariableState.UNUSEDVARIABLE:
+                component_property_path += self.UNUSED_VARIABLES
+            case VariableState.VARIABLE:
+                component_property_path += self.VARIABLES
+
+        component_property_path += variable_name + "|"
+        component_property_path += self.DATUM_PATH
+        component_property_path += variable_name
+
+        return component_property_path
+
+class ScriptCanvasComponent:
+
+    def __init__(self, editor_entity: EditorEntity):
+        self.script_canvas_component = editor_entity.add_component(SCRIPT_CANVAS_UI)
+
+    def get_script_canvas_component(self) -> EditorComponent:
+        return self.script_canvas_component
+
+    def set_component_graph_file_from_path(self, sc_file_path: str) -> None:
+        """
+        Function for updating the script canvas file being used by the component
+
+        param sc_file_path : the string path the new graph file on disk
+
+        returns None
+        """
+        sourcehandle = scriptcanvas.SourceHandleFromPath(sc_file_path)
+
+        self.script_canvas_component.set_component_property_value(Path.SCRIPT_CANVAS_ASSET_PROPERTY, sourcehandle)
+
+    def get_component_graph_file(self) -> str:
+        """
+        function for getting the component property value of the graph file in the script canvas component
+
+        returns the component property value of the script canvas source handle (string)
+        """
+
+        return self.script_canvas_component.get_component_property_value(Path.SCRIPT_CANVAS_COMPONENT_PROPERTY_PATH)
```

Comment:
Missed updating the enum in refactor from `SCRIPT_CANVAS_COMPONENT_PROPERTY_PATH` to `SCRIPT_CANVAS_ASSET_PROPERTY`

Reply:
fixed.